test/server/cards/04.5-AaN/AllAndNothing.spec.js

```diff
@@ -6,8 +6,8 @@ describe('All and Nothing', function() {
                     phase: 'conflict',
                     player1: {
                         inPlay: ['isawa-kaede', 'seeker-of-knowledge'],
-                        hand: ['all-and-nothing', 'all-and-nothing']
-                        // hand: ['all-and-nothing', 'all-and-nothing', 'taryu-jiai']
+                        // hand: ['all-and-nothing', 'all-and-nothing']
```

Comment:
Why not just remove the whole line?

Reply:
Yeah I noticed this after I setup the PR and didn't think it was worth adding another commit.  Just something I missed.  Completely agree that the commented out line should be removed.